ttnn/cpp/ttnn/operations/pool/generic/device/pool_op.hpp

```diff
@@ -7,26 +7,27 @@
 #include <optional>
 #include <variant>
 
-#include "ttnn/tensor/tensor.hpp"
 #include "ttnn/core.hpp"
-#include "ttnn/device_operation.hpp"
-#include "ttnn/types.hpp"
-#include "ttnn/operations/conv/conv2d/conv2d.hpp"
 #include "ttnn/cpp/ttnn/operations/sliding_window/sliding_window.hpp"
 #include "ttnn/decorators.hpp"
-
+#include "ttnn/device_operation.hpp"
+#include "ttnn/operations/conv/conv2d/conv2d.hpp"
```

Comment:
could you check whether these headers are really needed now or not??

Reply:
I found we are still calling functions like `conv::get_num_cores_nhw_from_parallel_config` in below, so this header might be still needed